src/components/UpdateVotes.jsx

```diff
@@ -0,0 +1,46 @@
+import icon from "../assets/like.jpg";
```

Comment:
Another option is to insert emojis just like any other text so you don't need any large image files, but that's totally up to you.